evansmusomi/react-native-102
# Adds Custom, Reusable Button

*What this does*
- Adds a custom reusable button

*How to verify*
- When you navigate to the auth screen
- You should see two custom buttons that look similar on Android and iOS
- The buttons can pass on custom props e.g. color and onPress handler
